src/util-decode-mime.c

```diff
@@ -1889,6 +1889,12 @@ static int ProcessMimeHeaders(const uint8_t *buf, uint32_t len,
                 state->found_child = 1;
                 entity->ctnt_flags |= CTNT_IS_MULTIPART;
 
+                if (*bptr == '"') {
+                    /* remove surrounding quotes */
+                    bptr++;
+                    blen -= 2;
```

Comment:
If FindMimeHeaderToken would return blen == 1 and the first byte is '"', would this result in the int underflow here?